fledging.qmd

```diff
@@ -6,7 +6,81 @@ Fledging is a goal of these JupyterHubs, so that users who have had success
+![](images/data-compute-cloud-grid.png){width="400" fig-align="center"}
+
+The proliferation of data centres to support cloud computing and data storage 
+has a [significant environmental cost](https://www.goclimate.com/knowledge/servers). 
+While most scientific workflows are small in comparison to other cloud uses 
+(e.g., AI and cryptocurrency mining), it is worth taking this into 
+consideration when thinking about "when to cloud".
+
+### Skills learned for working with **cloud-hosted data**
+
+The JupyterHubs provide a shared computing environment for mentors to learn and teach
+workflows for accessing and analyzing cloud-based data. Some of the skills we learn include:
+
+- Conceptual understanding of data in the cloud, and cloud-native data sources
+- Finding and accessing data in Earthdata cloud with earthaccess (Python) and 
+  earthdatalogin (R)
+- Streaming and subsetting cloud data with xarray (Python) and terra/rstac (R)
+- Authenticating with NASA Earthdata Login
+- Ability to ask questions, teach, and develop learning resources
+
+These skills enable efficient use of cloud-hosted data using either *local* or 
+*cloud* compute resources.
+
+### What are fledging paths for **compute**?
+
+Depending on computing needs and data size, there are many options for fledging.
+These may include:
+
+- Continue compute in the cloud using a shared environment (example: JupyterHub)
+    - If your institution offers a JupyterHub instance, this is a great option 
+      to seamlessly transfer your learnings from the Openscapes Hub to a Hub
+      for "real science".
+- Send large compute jobs to the cloud (example: Coiled)
+    - This is a good option for large compute jobs using Python that are easily 
+      parallelized (see an example 
+      [below](#an-example-path-for-computing-in-the-cloud)).
+- Compute locally with cloud-native workflows (example: streaming cloud data 
+  with earthaccess + xarray)
+    - For data and analysis jobs that can fit within your laptop's resrouces 
+      (RAM and hard disk), working locally with the same tools and workflows 
+      learned from working in the hub is a great option, and computing in the 
+      cloud is likely unnecessary. You can benefit greatly from the effiency of 
+      cloud data access patterns (such as streaming and range requests) without 
+      worrying about cloud costs and administration.
+- Compute locally using a shared environment “container” (example: Docker, 
+  Codespaces)
+    - One of the great benefits that the shared JupyterHub offers is a computing
+      environment that is already set up, so you don't have to go through the pain
+      of installing and configuring everything yourself. The images used in the 
+      JupyterHubs can often be used locally using tools like Docker. See some 
+      examples [here](https://pangeo-docker-images.readthedocs.io/en/latest/howto/launch.html#how-to-launch-jupyterlab-locally-with-one-of-these-images).
+
```

Comment:
I know jupyterlite doesn't support all of our packages yet, but I think it's an intriguing option!

Reply:
Definitely! I haven't followed its development closely, do you think it's worth mentioning here?